clusters/demo/base/kustomization.yaml

```diff
@@ -22,4 +22,5 @@ patches:
       annotationSelector: hmcts.github.com/kustomize-defaults != disabled
   - path: ../../../apps/toffee/demo/base/kustomize.yaml
   - path: ../../../apps/admin/demo/base/kustomize.yaml
-  - path: ../../../apps/admin/traefik-crds-upgrade-v27.0.2/kustomize.yaml
\ No newline at end of file
+  - path: ../../../apps/admin/traefik-crds-upgrade-v27.0.2/kustomize.yaml
+  - path: ../../../apps/dynatrace/base/kustomize-upgrade.yaml
```

Comment:
This file is still in main https://github.com/hmcts/sds-flux-config/blob/master/apps/dynatrace/base/kustomize-upgrade.yaml at the moment, not added as part of this PR